packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -1138,18 +1150,18 @@ describe('CheckoutModule', () => {
                     jest.resetAllMocks();
                 });
 
-                describe('when the user note exists in session storage', () => {
-                    it('should call dispatch with updateUserNote action and the user note', () => {
-                        // Arrange
-                        jest.spyOn(window.sessionStorage, 'getItem').mockReturnValue(userNote);
+                // describe('when the user note exists in session storage', () => {
+                //     it('should call dispatch with updateUserNote action and the user note', () => {
+                //         // Arrange
+                //         jest.spyOn(window.sessionStorage, 'getItem').mockReturnValue(userNote);
 
-                        // Act
-                        getUserNote(context);
+                //         // Act
+                //         getUserNote(context);
 
-                        // Assert
-                        expect(dispatch).toHaveBeenCalledWith('updateUserNote', userNote);
-                    });
-                });
+                //         // Assert
+                //         expect(dispatch).toHaveBeenCalledWith('updateUserNote', { type: 'delivery', note: userNote });
+                //     });
+                // });
```

Comment:
Remember to uncomment/remove these before merging :)

Reply:
Removed 🙂